### ISSUE TITLE
Use text links between login.html and signup.html

### ISSUE THREAD

**Issue by DevonJames** [dloa/publisher-web#63]

**Comment by DevonJames** [comment]
![log in or create account](https://cloud.githubusercontent.com/assets/7625406/25075109/44147ee8-22c1-11e7-82b4-1b1d8491a284.png)

**Comment by OstlerDev** [comment]
What if instead we added text to the circles that are already there?

**Comment by theryanjordan** [comment]
I think that is probably OK.  The only reason I changed it from the circles to the text links is that I struggled a bit trying to figure out how to "switch" over to the create new publisher view and couldn't figure it out.  Then of course I saw the blue circle and felt like a fool. I figured that if i struggled someone else might as well.  My only other concern with the circle is that there isn't one to switch back to log in once your on the create account view.  If you have an icon that communicates "log in" as well as the "new user", it's worth a try, but I can't think of a graphic that clearly communicates that, and I think both buttons/links should match size, placement, etc.
